### ISSUE TITLE
Add option to use hyphenation only when word cannot be displayed on a single line

### ISSUE THREAD

**Issue by joebez** [MrAsterisco/HyphenableText#1]
This package is great, but I'm looking for a way to only hyphenate words that do not fit on a single line. So shorter words like "fun-ny" would rarely, if ever, be hyphenated, but longer words like "whim-si-cal-ly" would be hyphenated more frequently.

**Comment by MrAsterisco** [comment]
Hey @joebez !
This is unfortunately not possible, unless we started building a dictionary of words that are likely to be hyphenated (but we'd have to do that possibly for all languages?).

The function we're using to determine where a word can be hyphenated (https://developer.apple.com/documentation/corefoundation/1542693-cfstringgethyphenationlocationbe) returns it for all words regardless.

Perhaps we could find a decent `length` under which we don't hyphen? For example, saying that words that are less than 5 characters should return immediately?

**Comment by joebez** [comment]
Hi @MrAsterisco. Yeah, I suspected it wouldn't be easy to do. I think your idea is a great one. Maybe `length` could be an optional parameter on the initializer. If `nil`, no words are skipped.

```
public init(_ text: String, ignoreWordsBelow length: Int? = nil) {
    self.text = text
    self.length = length
}
```

**Comment by joebez** [comment]
Actually, maybe just defining a minimum word length with a default length of zero would be better.

```
public init(_ text: String, ignoreWordsShorterThan minimumLength: Int = 0) {
    self.text = text
    self.minimumLength = minimumLength
}
```

**Comment by MrAsterisco** [comment]
@joebez Released version 1.0.2 with your change! 🥳 Thank you for contributing!
